modules/virt-about-resource-quota-limits.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-resource-quota-limits_{context}"]
+= About resource quota limits
+
+You can only set `ResourceQuotas` for virtual machines if the resource quota has a set of requests. If the resource quota contains both requests and limits, the request limits on the virtual machine must be set.
+
+Resource limits should be at least 100 megabytes larger than resource requests. For example:
+
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: with-limits
+spec:
+  running: false
+  template:
+    spec:
+      domain:
+        …
+        resources:
+          requests:
+            memory: 128Mi
+          limits:
+            memory: 256Mi  <1>
+----
+<1> `resources/limits` is twice the size of `resource/requests`
```

Comment:
I think this caveat can be misunderstood when written this way. Memory limits in general need to be larger than requests. currently that delta is about 100M and has slowly changed over time. In this example the limit is presented as twice the request, but to state that without context might imply there's a generalized rule to always double--which is not the case for larger requests.

Reply:
Hi @stu-gott - I had the same thought, lol. I have updated accordingly.